### DISCUSSION TITLE
Optional upload

### DISCUSSION THREAD

**Discussion by lucasantoniooficial** [rebing/graphql-laravel#1090] [Ideas]
The upload type recommended by the documentation requires the field.

It would be ideal if he left it optional by default and would only be obligated if he passed `Type::nonNull()` to him.

**Comment by mfn** [comment]
I don't follow, where are you having this issue? Can you provide a link to this library source what you mean?

**Reply by lucasantoniooficial** [reply]
I will create a use case to raise an issue.
